doc/api/errors.md

```diff
@@ -1213,25 +1213,6 @@ ongoing asynchronous operations.
 
 `c-ares` failed to set the DNS server.
 
-<a id="ERR_DOMAIN_CALLBACK_NOT_AVAILABLE"></a>
-
-### `ERR_DOMAIN_CALLBACK_NOT_AVAILABLE`
-
-The `node:domain` module was not usable since it could not establish the
-required error handling hooks, because
-[`process.setUncaughtExceptionCaptureCallback()`][] had been called at an
-earlier point in time.
-
-<a id="ERR_DOMAIN_CANNOT_SET_UNCAUGHT_EXCEPTION_CAPTURE"></a>
-
-### `ERR_DOMAIN_CANNOT_SET_UNCAUGHT_EXCEPTION_CAPTURE`
-
-[`process.setUncaughtExceptionCaptureCallback()`][] could not be called
-because the `node:domain` module has been loaded at an earlier point in time.
-
-The stack trace is extended to include the point in time at which the
-`node:domain` module had been loaded.
```

Comment:
This should be moved to https://nodejs.org/api/errors.html#legacy-nodejs-error-codes instead of being removed.